src/components/Settings/Settings.js

```diff
@@ -62,7 +32,19 @@ const Settings = () => {
       </div>
       <div className="settings__control">
         <h3>All addresses:</h3>
-        {addresses}
+        {isLoading && !error && <LoadingSpinner />}
+        {!isLoading && error && (
+          <Error>{error.message || 'Sth went wrong'}</Error>
+        )}
+        {!isLoading &&
+          !error &&
+          data.map((address) => (
```

Comment:
if you have only two properties you can do:
```suggestion
          data.map(({id, address}) => (
```

Reply:
Nicer way :smile: